foreman/data_refinery_foreman/foreman/main.py

```diff
@@ -799,11 +800,11 @@ def retry_failed_processor_jobs() -> None:
         pass
 
     failed_jobs = ProcessorJob.objects.filter(
-        success=False,
-        retried=False,
-        no_retry=False,
-        volume_index__in=active_volumes,
-        created_at__gt=JOB_CREATED_AT_CUTOFF
+        Q(success=False,
+          retried=False,
+          no_retry=False,
+          created_at__gt=JOB_CREATED_AT_CUTOFF) \
+        and (Q(volume_index__isnull=True) | Q(volume_index__in=active_volumes))
```

Comment:
Should we use `&` instead of `and`?

```suggestion
        & (Q(volume_index__isnull=True) | Q(volume_index__in=active_volumes))
```